source/adapters/level_zero/queue.cpp

```diff
@@ -896,8 +896,8 @@ ur_result_t urQueueFlush(
 
 ur_result_t urEnqueueKernelLaunchCustomExp(
     ur_queue_handle_t hQueue, ur_kernel_handle_t hKernel, uint32_t workDim,
-    const size_t *pGlobalWorkSize, const size_t *pLocalWorkSize,
-    uint32_t numPropsInLaunchPropList,
+    const size_t *pGlobalWorkOffset, const size_t *pGlobalWorkSize,
```

Comment:
You'll need to ignore `pGlobalWorkOffset` below.

Reply:
This is still not ignored, its causing build failures

Reply:
fixed